src/certificates/certificates-cards/CertificatesCards.test.jsx

```diff
@@ -0,0 +1,77 @@
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import initializeStore from '../../store';
+import { MODE_STATES } from '../data/constants';
+import CertificatesCards from './CertificatesCards';
+
+let store;
+
+const mockCertificates = [
```

Comment:
[question]: Do you think it’s worth putting this data into a separate file with mocks?